src/components/modules/toolbar/blockSettings.ts

```diff
@@ -195,7 +216,18 @@ export default class BlockSettings extends Module {
   private enableFlipper(): void {
     this.flipper = new Flipper({
       focusedItemClass: this.CSS.focusedButton,
-      activateCallback: () => {
+      activateCallback: (activatedItem) => {
+
+        /**
+         * If activated item is editable element, restore selection to block and flip to the next item
```

Comment:
can't understand this moment. Why do we need to restore selection to block, if, for example, an input at block settings is focused

Reply:
I thought about this scenario:
1. Open Block settings via tab
2. Type something to an editable element there
3. Press enter

What should happened after that?
For sure, Tune should do something with the typed text, but what about settings block. Should it be closed? At the moment caret is returned to the Block and next Tune is selected